app/experimenter/experiments/models/nimbus.py

```diff
@@ -458,51 +494,73 @@ def __str__(self):  # pragma: no cover
 
 
 class NimbusChangeLogManager(models.Manager):
+    STATUS_UNCHANGED = Q(old_status=F("new_status"))
+
+    STATUS_FROM_PREVIEW_TO_DRAFT = Q(
+        old_status=NimbusExperiment.Status.PREVIEW,
+        new_status=NimbusExperiment.Status.DRAFT,
+    )
+
+    PUBLISH_STATUS_FROM_IDLE_TO_REVIEW = Q(
+        old_publish_status=NimbusExperiment.PublishStatus.IDLE,
+        new_publish_status=NimbusExperiment.PublishStatus.REVIEW,
+    )
+
+    PUBLISH_STATUS_FROM_REVIEW_TO_APPROVED = Q(
+        old_publish_status=NimbusExperiment.PublishStatus.REVIEW,
+        new_publish_status=NimbusExperiment.PublishStatus.APPROVED,
+    )
+
+    PUBLISH_STATUS_FROM_WAITING_TO_REVIEW = Q(
+        old_publish_status=NimbusExperiment.PublishStatus.WAITING,
+        new_publish_status=NimbusExperiment.PublishStatus.REVIEW,
+    )
+
+    PUBLISH_STATUS_FROM_REVIEW_TO_IDLE = Q(
+        old_publish_status=NimbusExperiment.PublishStatus.REVIEW,
+        new_publish_status=NimbusExperiment.PublishStatus.IDLE,
+    )
+
+    PUBLISH_STATUS_FROM_WAITING_TO_IDLE = Q(
+        old_publish_status=NimbusExperiment.PublishStatus.WAITING,
+        new_publish_status=NimbusExperiment.PublishStatus.IDLE,
+    )
```

Comment:
Should these live alongside the [other ones](https://github.com/mozilla/experimenter/blob/main/app/experimenter/experiments/constants/nimbus.py#L199) in Nimbus constants?

Reply:
I stuck them here per [this advice over here](https://github.com/mozilla/experimenter/issues/5044#issuecomment-821355051):

> it occurs to me if we're going to have ones for different models they shoudln't live in NimbusConstants they should live right on the model

Reply:
Reworking to just use the latest changelog event makes all these queries obsolete